src/main/java/ch/njol/skript/lang/SkriptParser.java

```diff
@@ -251,6 +251,12 @@ public boolean hasTag(String tag) {
 
 							boolean success = element.init(parseResult.exprs, patternIndex, getParser().getHasDelayBefore(), parseResult);
 							if (success) {
+								for (Expression<?> expr : parseResult.exprs) {
+									if (!(expr instanceof UnparsedLiteral unparsedLiteral))
+										continue;
+									if (unparsedLiteral.multipleWarning())
+										break;
```

Comment:
this should just be 1 condition (plus comments are a good idea, and/or make this whole addition a helper method.)